### ISSUE TITLE
Issue with useInlineMode in Example

### ISSUE THREAD

**Issue by ryannichols** [Jeonhui/FittedSheets-SwiftUI#4]
I found this in my own code, then I downloaded the example app and tested there with the same results. When using the inline mode, if the parent view has an update, another sheet is added. It seems to be a problem where the FittedSheetPresenter doesn't check if the sheet has been added already, and will call the SheetController.animateIn() anyways. This just adds another instance of the same sheet. Can you help suggest how to fix this? There needs to be some way the view modifier knows it's already created the presenter and skip instantiating another one. Thanks!

**Comment by ryannichols** [comment]
This does the trick:
```
private func presentSheetController(_ sheetController: SheetViewController,
                                        parent viewController: UIViewController,
                                        useInlineMode: Bool) {
        if useInlineMode {
            if viewController.children.contains(where: { $0 is SheetViewController }) {
                return // Only one SheetViewController per view for inline sheets
            }
            if isPresented {
                DispatchQueue.main.async {
                    sheetController.animateIn(to: viewController.view, in: viewController)
                }
            } else {
                viewController.children.forEach { childViewController in
                    guard let svc = childViewController as? SheetViewController else { return }
                    DispatchQueue.main.async {
                        svc.animateOut()
                    }
                }
            }
        } else {
            if isPresented {
                viewController.present(sheetController, animated: animated)
            } else {
                viewController.presentedViewController?.dismiss(animated: animated)
            }
        }
    }
```

**Comment by Jeonhui** [comment]
@ryannichols 
This issue has been fixed and updated in v1.2.1.
Thanks to you, I was able to solve it quickly.
Can you check it works well on your app?

https://github.com/Jeonhui/FittedSheets-SwiftUI/releases/tag/1.2.1

**Comment by ryannichols** [comment]
Yes! That works great, thank you.
